zh/backup-to-aws-s3-using-br.md

```diff
@@ -186,7 +186,7 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` Custom Resource (CR)
         # checksum: true
         # sendCredToTikv: true
       from:
-        host: ${tidb_host}
+        host: ${tidb_host} # ${tidb_host} 为需要导出的 TiDB 的 service name，例如 basic-tidb
```

Comment:
No need to add a description for each of them, just add the description in L321.